package-lock.json

```diff
@@ -1,9647 +1,6 @@
 {
-    "name": "questionhub-prototype",
-    "lockfileVersion": 2,
     "requires": true,
-    "packages": {
-        "": {
-            "devDependencies": {
-                "@tailwindcss/forms": "^0.2.1",
-                "alpinejs": "^2.7.3",
-                "autoprefixer": "^10.1.0",
-                "axios": "^0.21",
-                "laravel-mix": "^6.0.6",
-                "lodash": "^4.17.19",
-                "postcss": "^8.2.1",
-                "postcss-import": "^12.0.1",
-                "tailwindcss": "^2.0.2"
```

Comment:
Please check if you have accidentally deleted all of these codes in your project folder. @mammubarak

Reply:
No i didnt  even touch , those files , ill check